src/components/pages/CompanyCertificates/CompanyCertificateCard.tsx

```diff
@@ -171,7 +171,7 @@ export default function CompanyCertificateCard({
           borderRadius: '74px',
         }}
         onClick={handleDelete}
-        disabled={!UserService.hasRole(ROLES.SUBSCRIBE_SERVICE_MARKETPLACE)}
+        disabled={!userHasPortalRole(ROLES.SUBSCRIBE_SERVICE)}
```

Comment:
that looks awkward. The user should get the "Subscribe Service" Role to access the Company Certificates? @oyo  lets quickly chat here later today

Reply:
@jjeroch changed this to DELETE_CERTIFICATES